Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM python:3.8
 WORKDIR /app
-RUN pip install numpy==1.19.*
-RUN pip install nltk==3.5.*
-RUN pip install fastapi[all]==0.61.*
+COPY requirements.txt .
```

Comment:
I had pulled these pip install commands out of requirements.txt and into the Dockerfile because I didn't want to confuse the competitors. The competitors will need to run requirements.txt on their local machines to make sure they have the libraries they need, but they do not need all the same libraries the server needs. For example, the competitors don't need fastapi. Is there a reason we should put these install commands in a separate .txt file rather than just embedding them in the Dockerfile? If so, maybe we should put them in a file with a different name, something like requirements_server.txt